scripts/BuildJSON/settings/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .dev import *
+
+CONNECTION_STRING = 'DRIVER={SQL Server};SERVER=(local);DATABASE=WRI;Trusted_Connection=yes;'
+JSONDIR = r'Z:\wri-web\src\jsonconfig'
```

Comment:
No, I don't think so because it will be different depending on the server it is deployed to.

I think that this file should just be ignored and include a `.sample`.